src/ucp/wireup/select.c

```diff
@@ -908,19 +881,20 @@ static inline int ucp_wireup_is_am_required(ucp_ep_h ep,
     return 0;
 }
 
-static ucs_status_t ucp_wireup_add_am_lane(ucp_ep_h ep, const ucp_ep_params_t *params,
-                                           unsigned ep_init_flags, unsigned address_count,
-                                           const ucp_address_entry_t *address_list,
-                                           ucp_wireup_lane_desc_t *lane_descs,
-                                           ucp_lane_index_t *num_lanes_p,
-                                           ucp_wireup_select_info_t *am_info)
+static ucs_status_t ucp_wireup_add_am_lane(ucp_wireup_select_ctx_t *select_ctx)
 {
+    ucp_ep_h ep                    = select_ctx->ep;
     ucp_wireup_criteria_t criteria = {0};
+    ucp_wireup_select_info_t *am_info;
+    uint64_t remote_cap_flags;
+    ucp_rsc_index_t dst_md_index;
     ucs_status_t status;
     int is_proxy;
 
-    if (!ucp_wireup_is_am_required(ep, params, ep_init_flags, lane_descs,
-                                   *num_lanes_p)) {
+    if (!ucp_wireup_is_am_required(ep, select_ctx->params,
```

Comment:
maybe pass 'const select_ctx' to ucp_wireup_is_am_required?

Reply:
done